test/CustomOperatorTest/TestOperator.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace CustomOperatorTest\Rx\Operator;
+
+use Rx\ObservableInterface;
+use Rx\ObserverInterface;
+use Rx\Operator\OperatorInterface;
+use Rx\SchedulerInterface;
+
+class TestOperator implements OperatorInterface
+{
+    private $mapTo;
+
+    public function __construct($mapTo)
+    {
+        $this->mapTo = $mapTo;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function __invoke(
+        ObservableInterface $observable,
+        ObserverInterface $observer,
+        SchedulerInterface $scheduler = null
+    ) {
+        return $observable
```

Comment:
This is a nit, but the `ObservableInterface` doesn't implement `mapTo`.  It might make sense to add an `AsObservable` class that converts anything that implements the `ObservableInterface` into an actual Observable.

This would become:
```PHP
  return (new AsObservable($observable))
            ->mapTo($this->mapTo)
            ->subscribe($observer);
```